packages/plugin/src/__tests__/compile.test.ts

```diff
@@ -1,5 +1,48 @@
-import { test } from 'vitest';
+import { expect, test } from "vitest";
```

Comment:
Looks like you're somehow overriding the eslint style preference for single quotes :).